README.md

```diff
@@ -7,7 +7,7 @@ Amazon SQS Connector allows you to connect to the Amazon Simple Queue Service (S
 ## Compatibility
 | Ballerina Language Versions | Amazon SQS API version  |
 | --------------------------- | ----------------------  |
-|            1.2.x            | 2012-11-05              |
+|            2.0.0            | 2012-11-05              |
```

Comment:
Please change it to the correct one. `Swan Lake Preview1`

Reply:
Changed.